ArduCopter/config.h

```diff
@@ -254,6 +254,11 @@
 #endif
 
 // max compass offset length (i.e. sqrt(offs_x^2+offs_y^2+offs_Z^2))
+#if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ALTERA
+ #ifndef COMPASS_OFFSETS_MAX
+  # define COMPASS_OFFSETS_MAX         1000  // OcPoC onboard compass has higher offsets
+ #endif
+#endif
```

Comment:
I've asked myself that very same question. The compass offsets are present both here and in the compass library.